docusaurus.config.js

```diff
@@ -98,11 +98,11 @@ const config = {
               },
               {
                 label: 'Discord',
-                href: 'https://discordapp.com/invite/docusaurus',
+                href: 'https://discord.com/invite/JZ2btyTfzQ',
```

Comment:
This change also closes #313 but you were assigned to #313. Please keep the PR's atomic (one change only). I'll merge this one.

Reply:
Sure, Thanks!